adapters/flipp/flipp.go

```diff
@@ -0,0 +1,225 @@
+package flipp
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+	"strings"
+
+	"github.com/buger/jsonparser"
+	"github.com/gofrs/uuid"
+	"github.com/prebid/openrtb/v17/openrtb2"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+const BannerType = "banner"
+const InlineDivName = "inline"
+const FlippBidder = "flipp"
+const DefaultCurrency = "USD"
+
+var Count = int64(1)
+var AdTypes = []int64{4309, 641}
+var DtxTypes = []int64{5061}
```

Comment:
nit:- These variable declarations can be grouped in a var block for better readability :-

```
var (
	Count    = int64(1)
	AdTypes  = []int64{4309, 641}
	DtxTypes = []int64{5061}
)
```